lesson2/task3.rb

```diff
@@ -0,0 +1,17 @@
+class ArraySelection
+
+  def initialize(my_arr)
+    @my_arr = my_arr
+  end
+
+  def even_selector
+    @my_arr.find_all { |elem| (str_to_i(elem) % 2) == 0 }
```

Comment:
Method `str_to_i ` can return nil. Please, handle it.

Reply:
if an array contains string `'a'`, method `str_to_i` receives `'a'`  and returns nil, because `item.to_s == item.to_s.to_i.to_s` returns `false`. So, a command `str_to_i('a') % 2` will raise an error.